apiserver/gui.go

```diff
@@ -286,13 +285,17 @@ func sendGUIComboFile(w io.Writer, fpath string) {
 
 // serveIndex serves the GUI index file.
 func (h *guiHandler) serveIndex(w http.ResponseWriter, req *http.Request) {
-	logger.Debugf("serving Juju Dashboard index")
+	logger.Tracef("serving Juju Dashboard index")
 	indexFile := filepath.Join(h.rootDir, "index.html")
-	t, err := template.ParseFiles(indexFile)
+
+	b, err := ioutil.ReadFile(indexFile)
```

Comment:
Not a template, so don't use the template methods, just read the file and write out. Could potentially use io.Copy, but didn't.